### PR TITLE
add fast-sync peer recommendation

### LINKED ISSUE #796: Document how best to run fast sync.
Mention that it's best to increase the number of minimum peers when running fast sync, particularly if fast sync is taking a long time to complete.

Recommend setting `--fast-sync-min-peers=8 ` in this case.

### PR DESCRIPTION
**Pull request by rolandtyler**
Signed-off-by: Roland Tyler <roland.tyler@consensys.net>

## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [x] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [ ] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [ ] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [ ] you fixed all the issues raised by the tests, if any.
- [ ] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change
Add new minimum number of peers to fast sync.

## Issue fixed
Fixes #796 

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

### CODE REVIEW

**Review by garyschulte — CHANGES_REQUESTED** [review]
LGTM, minor ask

File: docs/Concepts/Node-Types.md [hunk 1]
```diff
@@ -45,6 +45,11 @@ genesis block.
     the [`--network`](../Reference/CLI/CLI-Syntax.md#network) and [`--genesis-file`](../Reference/CLI/CLI-Syntax.md#genesis-file)
     options.
 
+!!! note
+
+    To improve performace, or if the fast sync is taking a long time to complete, use
+    [`fast-sync-min-peers`](../Reference/CLI/CLI-Syntax.md#fast-sync-min-peers) to increase the number of peers to at least 8.
```
Comment (alexandratran) [inline docs/Concepts/Node-Types.md:51]:
```suggestion
    the [`--fast-sync-min-peers`](../Reference/CLI/CLI-Syntax.md#fast-sync-min-peers) command line option to increase the number of peers to at least 8.
```

**Review by alexandratran — APPROVED** [review]
Minor suggestion, lgtm

File: docs/Concepts/Node-Types.md [hunk 2]
```diff
@@ -45,6 +45,11 @@ genesis block.
     the [`--network`](../Reference/CLI/CLI-Syntax.md#network) and [`--genesis-file`](../Reference/CLI/CLI-Syntax.md#genesis-file)
     options.
 
+!!! note
+
+    To improve performace, or if the fast sync is taking a long time to complete, use
```
Comment (bgravenorst) [inline docs/Concepts/Node-Types.md:50]: typo, "performance"

**Review by bgravenorst — APPROVED** [review]
Approved, but please fix the typo.

**Review by garyschulte — APPROVED** [review]
🚢

**Pull request merged**